src/libraries/Microsoft.PowerFx.Json/FormulaValueJSON.cs

```diff
@@ -163,8 +157,12 @@ internal static FormulaValue TableFromJsonArray(JsonElement array, TableType tab
         {
             Contract.Assert(array.ValueKind == JsonValueKind.Array);
 
-            var records = new List<RecordValue>();            
-            bool isArray = tableType?._type.IsColumn == true;
+            var records = new List<RecordValue>();
+
+            // Single Column table (e.g. [1,2,3]) Pattern for table is unique
+            // since in that case nested elements are not object and hence needs to be handled differently.
+            var nestedElementsAreObjects = array.EnumerateArray().Any(nestedElement => nestedElement.ValueKind == JsonValueKind.Object);

```

Comment:
>.Any( [](http://example.com/codeflow?start=65&length=5)

It would be good to have tests with mixed-type tables, something like `[1,{"a":2}]`. It should fail, but not throw.

Reply:
`nestedElementsAreObjects` is only used when the formulatype is single column table, and since single column table needs to be homogenous in Fx, I am not sure if I can test that scenario.